gpAux/gpperfmon/src/gpmon/gpmon_catqrynow.py

```diff
@@ -9,7 +9,11 @@
 #	print the line
 
 for line in open(os.path.join(GPMONDIR, "queries_now.dat")):
-    line = line.split('|')
+    line = line.strip().split('|')
```

Comment:
@KnightMurloc , why strip has been added here?

Reply:
to remove \n at the end of the line. Previously, this ended up in the array element and was overwritten, but now it interferes.